### PR TITLE
Small extensions

### PR DESCRIPTION
**Pull request by whaeck**
Addressing issues #100 and #101

### CODE REVIEW

File: python/src/tree/Tape.python.cpp [hunk 1]
```diff
@@ -5,7 +5,7 @@
 // local includes
 #include "ENDFtk/tree/Tape.hpp"
 #include "ENDFtk/tree/makeTape.hpp"
-#include "header-utilities/slurpFileToMemory.hpp"
+#include "ENDFtk/tree/fromFile.hpp"
```
Comment (nathangibson14) [inline python/src/tree/Tape.python.cpp:8]: Aren't you going to miss "slurp"?

**Review by nathangibson14 — CHANGES_REQUESTED** [review]
Shouldn't we be testing the `.front`?

Other than that, looks great.

File: python/src/tree/Tape.python.cpp (hunk 1 above)
Comment (whaeck) [reply python/src/tree/Tape.python.cpp:8]: Not in this lifetime ;-)

File: python/test/Test_ENDFtk_Tree_Tape.py [hunk 2]
```diff
@@ -221,6 +221,9 @@ def verify_material( self, material ) :
             material = materials.to_list()[0]
             verify_material( self, material )
 
+            material = materials.front()
+            verify_material( self, material )
```
Comment (nathangibson14) [inline python/test/Test_ENDFtk_Tree_Tape.py:225]: Do you think testing in Python but not C++ is sufficient?  I'm on the fence.

Comment (whaeck) [reply python/test/Test_ENDFtk_Tree_Tape.py:225]: This is only available in python, not on the C++ side.

Comment (nathangibson14) [reply python/test/Test_ENDFtk_Tree_Tape.py:225]: That's a good point.  `.front` works out of the box in C++. 🤦

**Review by nathangibson14 — APPROVED** [review]

**Pull request merged**